.github/workflows/code-lint.yaml

```
@@ -12,7 +12,7 @@ concurrency:
 
 jobs:
   docs:
-    uses: FuelLabs/github-actions/.github/workflows/vp-docs.yml@master
+    uses: FuelLabs/github-actions/.github/workflows/vp-docs.yml@ps/chore/relative-inline-snippets
```

```suggestion
    uses: FuelLabs/github-actions/.github/workflows/vp-docs.yml@master
```

After the review + merge of https://github.com/FuelLabs/github-actions/pull/48